htlcswitch/htlcnotifier.go

```diff
@@ -358,9 +362,10 @@ func (h *HtlcNotifier) NotifyForwardingFailEvent(key HtlcKey,
 // to as part of a forward or a receive to our node has been settled.
 //
 // Note this is part of the htlcNotifier interface.
-func (h *HtlcNotifier) NotifySettleEvent(key HtlcKey, eventType HtlcEventType) {
+func (h *HtlcNotifier) NotifySettleEvent(key HtlcKey, preimage lntypes.Preimage, eventType HtlcEventType) {
```

Comment:
nit: we try to wrap at 80 lines, so ideally this would be: 

```
func (h *HtlcNotifier) NotifySettleEvent(key HtlcKey,
     preimage lntypes.Preimage, eventType HtlcEventType)
```

Same applies in a few places in this PR.

Reply:
Changed in all placed.